src/util/symbols.js

```diff
@@ -1,10 +1,11 @@
-export const connected = '____skate_connected';
-export const created = '____skate_created';
+import createSymbol from './create-symbol';
+export const connected = createSymbol();
+export const created = createSymbol();
 export const name = '____skate_name';
```

Comment:
We do this instead: https://github.com/skatejs/skatejs/blob/master/src/api/vdom.js#L130. Only [`name` and `ref`](https://github.com/skatejs/skatejs/blob/master/src/api/vdom.js#L12) are used in that file and `ref` is used internally, and probably shouldn't actually be used outside as it should be expected that function helpers should be written with a specific version in mind. Cross-compat as far as API is concerned is limited to `name` so that you can use constructors for other versions because there's no spec'd way to do that.